tasties_app/models.py

```diff
@@ -1,3 +1,80 @@
-# from django.db import models
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator, MinLengthValidator, MaxValueValidator
+from datetime import datetime
 
-# Create your models here.
+
+class Category(models.Model):
+    category_name = models.CharField(max_length=16, unique=True)
+
+
+class Recipe(models.Model):
+    title = models.CharField(max_length=64, validators=[MinLengthValidator(1)], unique=True)
+    author_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    categories = models.ManyToManyField(Category)
+    description = models.CharField(max_length=512, validators=[MinLengthValidator(1)])
+    directions = models.CharField(max_length=65536, validators=[MinLengthValidator(1)])
+    publication_date = models.DateField(default=timezone.now)
+    minutes_to_make = models.IntegerField(validators=[MinValueValidator(1)])
+    recipe_picture = models.ImageField(upload_to='recipe_pictures')
+
+    def __str__(self):
+        return self.title
+
+
+class Ingredient(models.Model):
+    recipe_id = models.ForeignKey(Recipe, on_delete=models.CASCADE)
+    amount = models.DecimalField(max_digits=10, decimal_places=2, validators=[MinValueValidator(0.0001)])
+    measurement_unit = models.TextChoices('measurement_unit', 'ML FLOZ TSP OZ CUP GRAM WHOLE')
+    description = models.CharField(max_length=64, validators=[MinLengthValidator(1)])
+
+
+class Comment(models.Model):
+    author_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    recipe_id = models.ForeignKey(Recipe, on_delete=models.CASCADE)
+    publication_date = models.DateField(default=timezone.now, blank=True)
```

Comment:
blank=True - why did you make this change? Is it necessary? Notice that changing fields in the classes in models.py (but not methods) can and will lead to conflicts when we try to merge 0001_initial.py (initial migration).